setup.py

```diff
@@ -34,8 +32,10 @@ def find_version(version_file_path):
 extensions = []
 cmdclass = {}
 
-force_cuda = os.getenv("FORCE_CUDA", "0") == "1"
-if (torch.cuda.is_available() and CUDA_HOME is not None) or force_cuda:
+build_cuda_extensions = os.getenv("BUILD_CUDA_EXTENSIONS", "0") == "1"
```

Comment:
This is very nice! Can you also check whether it will download the latest version of torch during the pip install? Last time I checked, it always download and disregard the torch version I already have in my env.

Reply:
I wasn't clear, but I thought always downloading is not good for the users since some may have slow internet and downloading 1GB will take a long time for no good reason.